static/app/components/onboardingWizard/sidebar.tsx

```diff
@@ -243,4 +251,4 @@ const TopRight = styled('img')`
   width: 60%;
 `;
 
-export default withApi(withOrganization(withProjects(OnboardingWizardSidebar)));
```

Comment:
Do we still need those wrappers?

Reply:
I suspect it might be hard to fix tests without the wrappers, but maybe not. Let me verify.

`withOrganization` should be easy to swap to useOrganization. but `useProjects` sometimes be trickier